community/working-groups.md

```diff
@@ -0,0 +1,57 @@
+## Working Groups
+
+A working group is a group of people that are interested in working on a common
+topic. Working groups are informal and have no official authority. All working
+groups have a public channel on the [Julia Slack][slack]. Many working groups
+also hold regular meetings that are held online and are open to anyone. Meeting
+links are posted in the relevant Slack channels.
+
+The working group meetings are also listed on the
+[community events calendar][commmunity-events-calendar].
+
+### List of working groups
+
+Note: "ET" refers to Eastern Time.
```

Comment:
Use the "standard" UTC instead of the American-centric ET?

Reply:
So instead the british centric  UTC? We should do AOE instead

Reply:
I thought about converting all the times to UTC, but the challenge is that the offset between ET and UTC is not constant.

Reply:
I think the right way is to have it be turned by the browser into whatever time it is for the users

Reply:
Ok, then for the JuliaAstro meeting you can use whatever is the winter ET equivalent of 19:00 UTC

Reply:
No we should list that as 19:00 UTC,